test/controllers/close_reasons_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class CloseReasonsControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get close_reasons_index_url
+    assert_response :success
```

Comment:
You can also add `assert_not_nil assigns(:close_reasons)` to validate that `@close_reasons` has been assigned.